libs/sdk-backend-spi/src/organization/settings/index.ts

```diff
@@ -26,6 +26,24 @@ export interface IOrganizationSettingsService {
      */
     setLocale(locale: string): Promise<void>;
 
+    /**
+     * Sets timezone for organization.
+     *
+     * @param locale - for example "en-US", "cs-CZ", etc.
+     *
+     * @returns promise
+     */
+    setTimeZone(timezoneId: string): Promise<void>;
+
+    /**
+     * Sets date format for organization.
+     *
+     * @param formatId - example "en-US", "cs-CZ", etc.
```

Comment:
Please update this line accordingly. Especially the example, as it seems to be irrelevant.

Reply:
fixed